ajax/libs/emojionearea/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "emojionearea", 
+  "description": "EmojiOne Area is a small jQuery plugin that allows you to transform any html element into simple WYSIWYG editor with ability to use Emojione icons. The end result is a secure text/plain in which the image icons will be replaced with their Unicode analogues.",
+  "filename": "emojionearea.min.js",
+  "keywords": [
+	"emoji", 
+	"emojione", 
+	"picker", 
+	"emojionearea"
+  ],
+  "author": "Andrey Izman <izmanw@gmail.com>",
+  "homepage": "http://mervick.github.io/emojionearea/",
```

Comment:
Let's use `https` 
Using https is better because it's more security for browsing. To prevent the possibility the users been attacked during the redirection.
Thank you.